tests/chgres_cube/ftst_program_setup.F90

```diff
@@ -82,9 +82,63 @@ program ftst_program_setup
   if (my_rank .eq. 0) print*, "OK"
 
   ! Reading this namelist fails for some reason.
```

Comment:
It works now!

Reply:
As you found, the file did not have a newline at the end of it. GNU apparently insists that a namelist file end in a newline. Who am I to argue with the GNU Fortran compiler! ;-)

Reply:
Do you want to get rid of that comment?